SetFlags.cmake

```diff
@@ -159,14 +151,27 @@ function(set_exe_flags TARGET)
 
 			# Weverything with Clang exceptions:
 			-Weverything -Wno-error=disabled-macro-expansion -Wno-weak-vtables
-			-Wno-exit-time-destructors -Wno-string-conversion -Wno-c++98-compat-pedantic
-			-Wno-documentation -Wno-documentation-unknown-command -Wno-reserved-id-macro
-			-Wno-error=unused-command-line-argument
+			-Wno-global-constructors -Wno-exit-time-destructors -Wno-string-conversion
+			-Wno-c++98-compat-pedantic -Wno-documentation -Wno-documentation-unknown-command
+			-Wno-reserved-id-macro -Wno-error=unused-command-line-argument
```

Comment:
I'm loathe to pollute code with compiler flags and the warning-turned-error turns off language functionality that is clearly beneficial.

Reply:
The warning exists because global constructors are a potential "gotcha". Having to explicitly disable the warning just signifies that the author has thought about the issues and deemed it okay. It also increases visibility of the issue in code-review.